pkg/network/master/egressip.go

```diff
@@ -121,11 +108,11 @@ func (eim *egressIPManager) maybeDoUpdateEgressCIDRs() (bool, error) {
 		}
 	}
 
-	eim.monitorNodes = monitorNodes
+	eim.tracker.MonitorNodes = monitorNodes
 	if len(monitorNodes) > 0 {
 		if eim.stop == nil {
 			eim.stop = make(chan struct{})
-			go eim.poll(eim.stop)
+			go utilwait.PollUntil(common.DefaultPollInterval, eim.tracker.Poll, eim.stop)
```

Comment:
Here I am using the poll API that node (vxlan_monitor) defined. I thought it was cleaner utilizing the `utilwait` functionality as much as possible. I guess you're saying that `eim.poll` had a tighter retry loop than `evm.poll` did (0 sec vs. 1 sec), and that we should use that instead?

Reply:
no, I mean, why make the caller do the `utilwait.PollUtil` (and have to know the right `DefaultPollInterval` to use) rather than just having `eit.Poll` do that itself

Reply:
OK, that concern is handled by the public methods I described here: https://github.com/openshift/sdn/pull/291#discussion_r620981098 

Given the existence of a a public `AddEgressIP` function which `EgressIPTracker` would expose, we can mask `DefaultPollInterval` from the caller and keep it internal to `common`. I will re-push soon with a version of this.